### PR TITLE
Don't show upload failed notification for already deleted files

### PR DESCRIPTION
**Pull request by AndyScherzinger**
PR for #242 

@tobiasKaminsky please review since this can't be tested on an emulator :(  and instantUpload doesn't work on my phone...

### CODE REVIEW

File: res/values-fi/strings.xml [hunk 1]
```diff
@@ -20,7 +20,6 @@
   <string name="file_list_folders">kansiot</string>
   <string name="file_list_file">tiedosto</string>
   <string name="file_list_files">Tiedostot</string>
-  <string name="action_share_file">Jaa linkki</string>
```
Comment (MorrisJobke) [inline res/values-fi/strings.xml]: Please open a separate PR for unrelated changes :wink:

**Comment by AndyScherzinger** [comment]
@MorrisJobke please check - test on Morris' phone successful

**Comment by MorrisJobke** [comment]
LGTM

**Comment by tobiasKaminsky** [comment]
:+1: 
greetings from Leipzig ;-)
